src/scripts/cancel_older_approvals.py

```diff
@@ -0,0 +1,47 @@
+import os
+import requests
+
+CIRCLECI_TOKEN = os.getenv('CIRCLECI_TOKEN')
+CIRCLE_WORKFLOW_ID = os.getenv('CIRCLE_WORKFLOW_ID')
+CIRCLE_PROJECT_USERNAME = os.getenv('CIRCLE_PROJECT_USERNAME')
+VCS="gh"
+CIRCLE_PROJECT_REPONAME = os.getenv('CIRCLE_PROJECT_REPONAME')
+BASE_URL="https://circleci.com/api/v2"
+headers = {'Circle-Token': CIRCLECI_TOKEN}
+
+current_wf_url = f'{BASE_URL}/workflow/{CIRCLE_WORKFLOW_ID}'
+
+
+try:
+    current_workflow = requests.get(current_wf_url).json()
```

Comment:
It would be better if we wrapped all request calls with retry logic given how fragile we've found them to be.